core/wasm/runtest/src/lib.rs

```diff
@@ -291,6 +307,27 @@ mod tests {
         };
     }
 
+    #[test]
+    fn test_studio_total_supply()  {
+        let input_data = b"{}";
+
+        let outcome = run_with_filename(
+            b"totalSupply",
+            input_data,
+            &[],
+            &runtime_context(0, 0, 0),
+            "res/studio.wasm",
+        ).expect("ok");
+
+        println!("Gas used for simple call {}", outcome.gas_used);
+
+        match outcome.return_data {
```

Comment:
You could do `outcome.return_data.expect("expected returned value")` to avoid match

Reply:
Still need to match the ReturnData::Value so let's keep it for now